app.test.js

```diff
@@ -134,3 +135,57 @@ test('should respond with a 400 if the provided id is not valid (eg not a number
 })
 })
 
+describe('GET /api/articles/:article_id/comments',()=>{
+    test('Should respond with 200 and the correct comments for the article',()=>{
```

Comment:
test description could be improved here as you are really checking the default sort/order